src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -292,6 +292,9 @@ protected void setup()
 
         SSTableHeaderFix.fixNonFrozenUDTIfUpgradeFrom30();
 
+        StorageService.instance.snapshotManager.loadSnapshots();
```

Comment:
Moved before scrubbing. We are ever going to load snapshots (scanning the dirs) just once.